clear-intermediate-datasets/python-runnables/clear_int_datasets/runnable.py

```diff
@@ -88,7 +94,25 @@ def run(self, progress_callback):
         logging.info("Found {} PARTITIONED datasets: {}".format(str(len(partd_datasets)),
                                                                 str(partd_datasets)))
 
-        # List all datasets to keep, potentially including shared & partd ones:
+        # Add dataset types to results list
+        results = []
+
+        for obj in flow_inputs:
```

Comment:
prefer explicit variable names, e.g, obj => dataset , flow_inputs => input_datasets

Reply:
Done in commit [892642f55ab9f78b502aec620fc31c0dc77d448c](https://github.com/gbetegon88/dataiku-contrib/commit/892642f55ab9f78b502aec620fc31c0dc77d448c#diff-6f25a7eaa37b1d34604450d8bc9334c0558917e69a3eb97e69bea9d70378e561).

I didn't change flow_input or flow_ouput bc I think is more clear than 'input_dataset' (a dataset can be an input to a recipe but not a flow input)